rwiki/rwiki-impl/impl/src/java/uk/ac/cam/caret/sakai/rwiki/component/service/impl/RWikiObjectServiceImpl.java

```diff
@@ -422,6 +465,15 @@ private void update(String name, String user, String realm, Date version,
 		RWikiCurrentObject rwo = getRWikiObject(name, realm);
 		RWikiHistoryObject rwho = null;
 
+		String groupsString = "";
+		if (groups != null) {
+			boolean isFirstGroup = true;
+			for (String group : groups) {
+				groupsString += (isFirstGroup? "" : "," ) + group;
```

Comment:
Maybe we can use the StringJoiner class?

Reply:
Ok